packages/component/src/Localization/nl-NL.js

```diff
@@ -1,23 +1,71 @@
+function botSaidSomething(avatarInitials, text, timestamp) {
+  return `Bot ${avatarInitials} zei; ${text}, ${xMinutesAgo(timestamp)}`;
+}
+
+function userSaidSomething(avatarInitials, text, timestamp) {
+  return `Gebruiker ${avatarInitials} zei; ${text}, ${xMinutesAgo(timestamp)}`;
+}
+
+function xMinutesAgo(dateStr) {
+  const date = new Date(dateStr);
+  const dateTime = date.getTime();
+
+  if (isNaN(dateTime)) {
+    return dateStr;
+  }
+
+  const now = Date.now();
+  const deltaInMs = now - dateTime;
+  const deltaInMinutes = Math.floor(deltaInMs / 60000);
+  const deltaInHours = Math.floor(deltaInMs / 3600000);
+
+  if (deltaInMinutes < 1) {
+    return 'Zojuist';
+  } else if (deltaInMinutes === 1) {
+    return 'Een minuut geleden';
+  } else if (deltaInHours < 1) {
+    return `${deltaInMinutes} minuten geleden`;
+  } else if (deltaInHours === 1) {
+    return `Een uur geleden`;
+  } else if (deltaInHours < 5) {
+    return `${deltaInHours} uur geleden`;
+  } else if (deltaInHours <= 24) {
+    return `Vandaag`;
+  } else if (deltaInHours <= 48) {
+    return `Gisteren`;
+  } else if (window.Intl) {
+    return new Intl.DateTimeFormat('nl-NL').format(date);
+  } else {
+    return date.toLocaleString('nl-NL', {
+      day: '2-digit',
+      hour: '2-digit',
+      hour12: false,
+      minute: '2-digit',
+      month: '2-digit',
+      year: 'numeric',
+    });
+  }
+}
+
 export default {
-  // FAILED_CONNECTION_NOTIFICATION: '',
-  // Do not localize {Retry}; it is a placeholder for "Retry". English translation should be, "Send failed. Retry."
-  SEND_FAILED_KEY: 'versturen mislukt, {Retry}.',
-  // SLOW_CONNECTION_NOTIFICATION: '',
+  FAILED_CONNECTION_NOTIFICATION: 'Verbinding maken niet mogelijk.',
+  SEND_FAILED_KEY: 'Versturen mislukt, {retry}.',
```

Comment:
In English, the phrase "Send failed. Retry." is grammatically more correct than "Send failed, retry". In Dutch, is there a preference in this case?

Reply:
if "Versturen mislukt. Opnieuw proberen" makes more sense, could you update this string to the following: 

`Versturen mislukt. {Retry}.'`

Reply:
`"Versturen mislukt, probeer opnieuw."` would be my preference. Will discuss this with another Dutch colleague tomorrow, but in Dutch we don't prefer cutting up those sentences.

Reply:
Once I get confirmation from you that the single sentence is preferred, I will merge this PR in. Thanks for all your help!

Reply:
Confirmed. @corinagum